### PR TITLE
Issue #4141: Split and Organize Checkstyle inputs by Test for ConstantName

### PR DESCRIPTION
**Pull request by Kietzmann**
Related to #4141

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/constantname/InputConstantNames.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.puppycrawl.tools.checkstyle.checks.naming;
+package com.puppycrawl.tools.checkstyle.checks.naming.constantname;
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/constantname/InputConstantNames.java]: Can we rename this file/class to just `InputConstantName`, no `s` on end.

Comment (Kietzmann) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/constantname/InputConstantNames.java]: Done

**Comment by romani** [comment]
CIs should be green, please rebase on our latest master to resolve TC issue

**Comment by Kietzmann** [comment]
Done

**Comment by romani** [comment]
please rebase on our latest master to fix werker. TC failure due to license could be ignored, I reported and issue to their support.

**Comment by Kietzmann** [comment]
@romani 
Done
Seems that travis failed at different stages running the same code
Don't you think that is quite strange behavior?

**Comment by rnveach** [comment]
Travis was the "VM crashed" error. I restarted it and it passed.
TC is a timeout error. I don't have authority to restart it.

**Comment by Kietzmann** [comment]
@rnveach 
Thanks :)

**Pull request merged**